tf2_ros/src/buffer_client.cpp

```diff
@@ -40,7 +40,6 @@ namespace tf2_ros
 {
   BufferClient::BufferClient(std::string ns, double check_frequency, ros::Duration timeout_padding): 
     client_(ns), 
-    check_frequency_(check_frequency),
```

Comment:
If we keep the member for compatibility we should still initialize it to avoid warnings.